packages/suite/src/components/settings/DeviceBanner.tsx

```diff
@@ -12,13 +12,10 @@ const StyledLottieAnimation = styled(LottieAnimation)`
     background: ${({ theme }) => theme.BG_GREY};
 `;
 
-const Wrapper = styled.div`
+const Wrapper = styled(Card)`
```

Comment:
nit: can simply be:

```
const Wrapper = styled(Card)`
    flex-direction: row;
`;
```

Reply:
c6c0d40ef00dd874f2c611892cb1e0bacc7992d0